tgui/packages/tgui-say/handlers/incrementChannel.tsx

```diff
@@ -1,32 +1,48 @@
 import { CHANNELS } from '../constants';
 import { Modal } from '../types';
+// Insert the names of channels you want to not cycle on tab here
+const BLACKLIST = ['Admin'];
+const BLACKLISTED_CHANNEL_INDICES = CHANNELS.map((channel, index) => {
```

Comment:
Nit: We have `filterMap` for this in collections.ts

Reply:
Filtermap requires having not readonly

Reply:
It shouldn't if you do something like:

`filterMap(CHANNELS, (channel: readonly string, index)` or something. What error were you getting with what code?

Reply:
Linter says "Argument of type 'readonly ["Say", "Radio", "Me", "OOC", "Admin"]' is not assignable to parameter of type 'unknown[]'.
  The type 'readonly ["Say", "Radio", "Me", "OOC", "Admin"]' is 'readonly' and cannot be assigned to the mutable type 'unknown[]'.". The code compiles, but doesnt work. Trying to get it working only caused more problems, since afaik filtermap doesnt use two argument lambda. I think that simplicity of using filter and map separately is good enough here.

Reply:
Dang, oh well